spec/features/visitor_can_delete_a_cart_item_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.